networkit/cpp/community/test/CommunityGTest.cpp

```diff
@@ -688,6 +704,11 @@ TEST_F(CommunityGTest, testCoverF1Similarity) {
     const double pre = 2.0 / 11.0;
     const double re = 2.0 / 10.0;
     const double f1 = 2.0 * (pre * re) / (pre + re);
+    std::vector<double> resVector=sim.getValues();
+    EXPECT_EQ(3, resVector.size());
+    EXPECT_DOUBLE_EQ(1.0, resVector[0]);
+    EXPECT_DOUBLE_EQ(f1, resVector[1]);
+    EXPECT_DOUBLE_EQ(0.0, resVector[2]);
```

Comment:
Thanks for your Suggestions! I already included all of them except for this, because EXPECT_THAT is not defined in the "gtest.h" yet, shall we include it aswell? Best regards

Reply:
Yes, you can use `EXPECT_THAT` if you add `#include <gmock/gmock.h>` here is an example: https://github.com/networkit/networkit/blob/master/networkit/cpp/centrality/test/TopClosenessGTest.cpp
Thanks!